manifests/l/LiteratureAndLatte/Scapple/1.4.2.0/LiteratureAndLatte.Scapple.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with komac v2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: LiteratureAndLatte.Scapple
+PackageVersion: 1.4.2.0
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2022-12-09
+InstallationMetadata:
+  DefaultInstallLocation: '%PROGRAMFILES%\Scapple'
+InstallerSwitches:
+  Silent: --unattendedmodeui none --mode unattended
+  SilentWithProgress: --unattendedmodeui minimalWithDialogs --mode unattended
+  Log: --debugtrace <LOGPATH>
+  InstallLocation: --prefix <INSTALLPATH>
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerUrl: https://scrivener.s3.amazonaws.com/Scapple-installer.exe
```

```suggestion
  InstallerUrl: https://www.literatureandlatte.com/download_action.php?url=https://scrivener.s3.amazonaws.com/Scapple-installer.exe
```

Further investigation needed.